.snippets/text/get-started/fine-tuning-supported-models.md

```diff
@@ -1,2 +1,2 @@
-- [klusterai/Meta-Llama-3.1-8B-Instruct-Turbo](/get-started/models/){target=_blank}
-- [klusterai/Meta-Llama-3.3-70B-Instruct-Turbo](/get-started/models/){target=_blank} 
\ No newline at end of file
+- [klusterai/Meta-Llama-3.1-8B-Instruct-Turbo](/get-started/models/){target=_blank}.
+- [klusterai/Meta-Llama-3.3-70B-Instruct-Turbo](/get-started/models/){target=_blank}.
```

Comment:
We don't need punctuation here. Reference: https://developers.google.com/style/lists?hl=en